.gitignore

```diff
@@ -70,3 +70,8 @@ __pycache__/
 /.settings/
 /.cproject
 /.project
+.Rproj.user
+
+.Rhistory
+
+panda3d.Rproj
```

Comment:
Did you commit these changes intentionally?  Could you explain them if so?

Reply:
That was some information that snuck in when I was playing around with using R as my interface for coding and interacting with git. I have since learned better ways to do this using VSCode and GitKraken